### DISCUSSION TITLE
Welcome to readable_climate_reports Discussions!

### DISCUSSION THREAD

**Discussion by mitko** [mitko/readable_climate_reports#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect and coordinate. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded.

**Comment by petermr** [comment]
Suggest we create a channel for introducing ourselves (for those who are happy to).

**Reply by mitko** [reply]
Yes! Do you want to drive this? I'm happy to share any info about myself, background, etc.

**Reply by petermr** [reply]
sure
